pyproject.toml

```diff
@@ -99,6 +96,11 @@ dev = [
   "uvloop; platform_system != 'Windows'",
   "grpc-interceptor[testing]",
 ]
+umap = [
```

Comment:
Maybe embeddings?

Reply:
Also need to add this in the docker build for now

Reply:
done